tablo/interfaces/arcgis/views.py

```diff
@@ -212,13 +212,15 @@ def handle_request(self, request, **kwargs):
 
 
 class QueryView(FeatureLayerView):
-    query_limit_default = 50000
+    query_limit_default = 1000
```

Comment:
I was saying, since you need the same value for the maxRecordCount entries, you should pull that up into a constant rather that having `maxRecordCount: 10000` in two places, plus your query_limit_default value.